lms/assets.ini

```diff
@@ -1,14 +1,8 @@
 [bundles]
 
-file_picker_js =
-  scripts/file_picker.bundle.js
```

Comment:
When these dependencies are removed, the following frontend dependencies can also be removed:

- lodash (replaced by JS standard lib APIs)
- jquery (replaced by Preact + DOM APIs)
- superagent (replaced by whatwg-fetch)

I can do that in a follow up PR as some small changes will also be needed in the gulpfile

Reply:
Cool, thanks.

Now that I think of it we'll probably have some unused backend deps now too